tests/e2e/features/artifacts/whereToSendPage/artifacts.feature

```diff
@@ -0,0 +1,80 @@
+@artifacts @regression @whereToSendPage @emptyWallet @various
+Feature: Artifacts - UI
+
+  Background:
+    Given Connect Metamask extension with login action
+
+  @id1560
+  Scenario: Check artifacts for an "Where to send" page (Testnet)
```

Comment:
I would recommend to split it for id1560:I  and id1560:II and use `Scenario Outline`
The first one will contain just
```
When I go to page "/transaction/zksync/era/?network=era-goerli" # as we have Given in the Background
Then Element with "<typeID>" "<value>" should be "<flag>"
```
For a help you may use an example of id1430:I

and the 2nd one part is:

```
When I go to page "/transaction/zksync/era/?network=era-goerli" # as we have Given in the Background
Then Arrow element for "<value>" external link should be "<flag>"
```